README.md

```diff
@@ -18,59 +18,120 @@ UCX leverages Databricks Lakehouse platform to upgrade itself. The upgrade proce
+- Select a workspace profile that has been defined in `~/.databrickscfg`
+- Provide the name of the inventory database where UCX will store the assessment results. This will be in the workspace `hive_metastore`. Defaults to `ucx`
+- Create a new or select an existing SQL warehouse to run assessment dashboards on. The existing warehouse must be Pro or Serverless.
+- Configurations for workspace local groups migration:
+  - Provide a backup prefix. This is used to rename workspace local groups after they have been migrated. Defaults to `db-temp-`
+  - Select a workspace local groups migration strategy. UCX offers matching by name or external ID, using a prefix/suffix, or using regex.
+  - Provide a specific list of workspace local groups (or all groups) to be migrated.
+- Select a Python log level, e.g. `DEBUG`, `INFO`. Defaults to `INFO`
+- Provide the level of parallelism, which limit the number of concurrent operation as UCX scans the workspace. Defaults to 8.
+- Select whether UCX should connect to the external HMS, if a cluster policy with external HMS is detected. Defaults to no.
+
+After this, UCX will be installed locally and a number of assets will be deployed in the selected workspace. These assets are available under the installation folder, i.e. `/Users/<your user>/.ucx/`
+
 ![macos_install_ucx](docs/macos_2_databrickslabsmac_installucx.gif)
 
 #### Upgrade UCX
+Verify that UCX is installed
+```text
+databricks labs installed
+
+Name  Description                            Version
+ucx   Unity Catalog Migration Toolkit (UCX)  <version>
+```
+Upgrade UCX via Databricks CLI:
+```commandline
+databricks labs upgrade ucx
+```
+The prompts will be similar to [Installation](#install-ucx)
+
 ![macos_upgrade_ucx](docs/macos_3_databrickslabsmac_upgradeucx.gif)
 
 #### Uninstall UCX
+Uninstall UCX via Databricks CLI:
+```commandline
+databricks labs uninstall ucx
+```
+
+Databricks CLI will confirm a few options:
+- Whether you want to remove all ucx artefacts from the workspace as well. Defaults to no.
+- Whether you want to delete the inventory database in `hive_metastore`. Defaults to no.
+
 ![macos_uninstall_ucx](docs/macos_4_databrickslabsmac_uninstallucx.gif)
 
+### Using UCX
+
+After installation, a number of UCX workflows will be available in the workspace. `<installation_path>/README` contains further instructions and explanations of these workflows.
+UCX also provides a number of command line utilities accessible via `databricks labs ucx`.
+
+#### Understanding assessment report
+
+After UCX assessment workflow is executed, the assessment dashboard will be populated with findings and common recommendations.
+[This guide](docs/assessment.md) talks about them in more details.
+
+#### Synchronising workspace info
+Use to upload workspace config to all workspaces in the account where UCX is installed. UCX will prompt you to select an account profile that has been defined in `~/.databrickscfg`
```

Comment:
What this will allow the user to do ? Is it a UCX cross-workspace installation ?